pkg/controller/manager/manager_controller.go

```diff
@@ -305,6 +305,10 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		}
 	}
 
+	if certOperatorManaged && installation.CertificateManagement != nil {
+		tlsSecret = nil
```

Comment:
I do not see where this secret is being used after this. Why do we need to se this to nil ?